server/chat-commands.js

```diff
@@ -4173,6 +4186,7 @@ const commands = {
 				avatar: targetUser.avatar,
 				group: targetUser.group,
 				autoconfirmed: !!targetUser.autoconfirmed,
+				status: targetUser.getStatus(),
```

Comment:
```suggestion
				status: targetUser.status,
				away: targetUser.away,
```

I need this to make Zarel/Pokemon-Showdown-Client#1265 work